nova_vm/src/ecmascript/abstract_operations/testing_and_comparison.rs

```diff
@@ -277,3 +277,133 @@ pub(crate) fn is_less_than<const LEFT_FIRST: bool>(
         Ok(Some(rnx < rny))
     }
 }
+
+/// [7.2.14 IsLooselyEqual ( x, y )](https://tc39.es/ecma262/#sec-islooselyequal)
+///
+/// The abstract operation IsLooselyEqual takes arguments x (an ECMAScript
+/// language value) and y (an ECMAScript language value) and returns either a
+/// normal completion containing a Boolean or a throw completion. It provides
+/// the semantics for the == operator. It performs the following steps when
```

Comment:
**nitpick:** I've been leaving out the `It performs the following steps when called:` part out of the doc comments since we do not actually write out the steps but instead implement them :)